BankAPILibrary/GiniBankAPILibrary/Sources/GiniBankAPILibrary/Documents/Document.swift

```diff
@@ -46,6 +46,30 @@ public struct Document {
         case progress
         case sourceClassification
     }
+    
+    public init(compositeDocuments: [CompositeDocument]?,
```

Comment:
Except `id` all other fields can have default values so that clients can call `Document(id: "1234-abcd")` to create a document.

You should check which fields other than `id` we use in the Capture and Bank SDKs.